src/index.ts

```diff
@@ -79,26 +79,58 @@ export {
  */
 export async function createVerifiableCredentialJwt(
   payload: JwtCredentialPayload | CredentialPayload,
-  issuer: Issuer,
+  issuer: Issuer | Issuer[],
   options: CreateCredentialOptions = {}
 ): Promise<JWT> {
   const parsedPayload: JwtCredentialPayload = {
     iat: undefined,
     ...transformCredentialInput(payload, options.removeOriginalFields),
   }
   validateJwtCredentialPayload(parsedPayload)
-  return createJWT(
-    parsedPayload,
-    {
-      ...options,
-      issuer: issuer.did || parsedPayload.iss || '',
-      signer: issuer.signer,
-    },
-    {
-      ...options.header,
-      alg: issuer.alg || options.header?.alg || JWT_ALG,
+
+  if (!Array.isArray(issuer)) {
+    return createJWT(
+      parsedPayload,
+      {
+        ...options,
+        issuer: issuer.did || parsedPayload.iss || '',
+        signer: issuer.signer,
+      },
+      {
+        ...options.header,
+        alg: issuer.alg || options.header?.alg || JWT_ALG,
+      }
+    )
+  } else {
+    const did = issuer[0].did
+    const issuers = []
+    for (const iss of issuer) {
+      if (iss.did !== did) {
+        throw new Error('All issuers must be the same did to comply with the Verifiable Conditions spec')
+      }
+      issuers.push({
+        issuer: iss.did || parsedPayload.iss || '',
+        signer: iss.signer,
+        alg: iss.alg || options.header?.alg || JWT_ALG,
+      })
     }
-  )
+
+    return createMultisignatureJWT(parsedPayload, { ...options }, issuers)
+  }
+}
+
+/**
+ * Adds an additional signature to an already signed VerifiableCredential 
+ * given a `jwt` and an `Issuer`.
+ *
+ * @param {JWT} jwt - The JWT to add a new signature to
+ * @param {Issuer} issuer - The issuer to use to sign the JWT
+ *
+ * @returns {Promise<JWT>} - The JWT with the new signature added
+ */
+export async function addSignatureToJwt(jwt: JWT, issuer: Issuer): Promise<JWT> {
```

Comment:
Please remove this since it's not implemented.

Reply:
done